### ISSUE TITLE
Device found, but can't retrieve Package list, error connecting to service.

### ISSUE THREAD

**Issue by Megaddd** [google/gapid#1827]
[Getting the spinning donut of doom whenever I try to select the 'Package / Action'](https://i.imgur.com/nBKcmo7.png)
GAPID log:
```
I 04:02:49.128 gapic Looking for GAPID in C:\Program Files (x86)\gapid -> C:\Program Files (x86)\gapid\gapis.exe
I 04:02:49.147 gapic Starting gapis as [C:\Program Files (x86)\gapid\gapis.exe, -enable-local-files, -crashreport, -analytics, 761ad99f-0d1d-4e6a-8270-f07eec001e79, -log-file, C:\Users\Megaddd\AppData\Local\Temp\gapis.log, -log-level, Info, -gapir-args, --log C:\\Users\\Megaddd\\AppData\\Local\\Temp\\gapir.log --log-level I, --strings, C:\Program Files (x86)\gapid\strings, --gapis-auth-token, I1az9cUv, --idle-timeout, 60000ms, --adb, H:\Android\android-sdk\platform-tools\adb.exe]
I 04:02:49.187 gapic gapis: 13:02:49.187 I: <gapis> Logging to: C:\Users\Megaddd\AppData\Local\Temp\gapis.log
I 04:02:49.339 gapic gapis: Bound on port '54562'
I 04:02:49.340 gapic Detected gapis startup on port 54562
I 04:02:49.341 gapic Established a new client connection to 54562
I 04:02:49.375 gapic [ManagedChannelImpl@65c53ffa] Created with target localhost:54562
I 04:02:49.474 gapic Server info: name: "MEGADDD_GRANDY" version_major: 1 version_point: 3
I 04:02:49.664 gapis Examining gapid.apk on host...
I 04:02:49.667 gapis Looking for gapid.apk...
I 04:02:50.491 gapis Found gapid package...
I 04:02:51.312 gapis Adding new device
I 04:02:51.312 gapis Adding new device
I 04:02:51.312 gapis New scheduler for device: a3d29f85684ac431fdf586757dac7f2075518281
I 04:03:00.561 gapic Starting gapit as [C:\Program Files (x86)\gapid\gapit.exe, -analytics, 761ad99f-0d1d-4e6a-8270-f07eec001e79, packages, --format, proto, --icons, --icondensity, 1.0, --dataheader, ~-~-~-~-, --device, 41003c6d446771cf, --adb, H:\Android\android-sdk\platform-tools\adb.exe]
W 04:03:33.456 gapic gapit: 13:03:33.456 E: <gapit> Main failed
W 04:03:33.456 gapic gapit: Error: getting package list
W 04:03:33.457 gapic gapit:    Cause: Connecting to service port
W 04:03:33.457 gapic gapit:    Cause: 
W 04:03:33.457 gapic gapit:    Cause: Timeout connecting to service
W 04:03:33.518 gapic Failed to read package info
I 04:03:33.521 gapic gapit exited cleanly
I 04:03:33.522 gapic Shutting down gapit
I 04:03:34.625 gapic Crash Report Uploaded Successfully; Crash Report ID: ccb096d18c08d35e

```

**Comment by ben-clayton** [comment]
Hi @Megaddd,
I'm sorry you're having this issue. To help us figure out what's going wrong, please can you:

1. Give the Android device model.
2. Give the Android OS version / build?
3. Look in the output of `adb logcat` while clicking on the `...` button for an error or crash relating to `PackageInfoService` ? If there's a stack trace or message, please copy that into this bug.

Many thanks,
Ben

**Comment by Megaddd** [comment]
1. Samsung Galaxy Alpha
2. 5.0.2 / LRX22G.G850FXXU2COI3
3. Don't know about any `. . .` button, but outputting logcat to a txt file I found two instances of ![java.util.concurrent.ExecutionException: android.content.res.Resources$NotFoundException: Unable to find resource ID #0x7f020002](https://pastebin.com/jGZcz3iF)

**Comment by AWoloszyn** [comment]
This might be fixed in 1.2.0. Please re-open if you still see it.
